src/PrestaShopBundle/Resources/config/services/core/grid/doctrine_query_builder.yml

```diff
@@ -438,6 +438,14 @@ services:
       - "@=service('prestashop.adapter.legacy.context').getContext().language.id"
     public: true
 
+  prestashop.core.grid.query.builder.customer_thread:
```

Comment:
I think you can use FQCN ? https://github.com/PrestaShop/ADR/blob/master/0019-FQCN-Autowiring.md

Reply:
My merge went wrong I need to rebase I think